integration/fuse/src/test/java/alluxio/fuse/AlluxioJniFuseFileSystemTest.java

```diff
@@ -512,7 +514,14 @@ public void rmdir() throws Exception {
 
   @Test
   public void write() throws Exception {
-    FileOutStream fos = mock(FileOutStream.class);
+    FileOutStream fos = spy(new FileOutStream() {
+      @Override
+      public void write(int b) throws IOException {
+        // although b is int, the 24 higher bits are ignored
+        // we are effectively writing 1 byte
+        mBytesWritten += 1;
+      }
```

Comment:
Mocking without stubbing any methods will create a mock that returns the default value of the return type. In this case `FileOutStream#getBytesWritten` will always return 0. To properly account for the number of bytes written into the underlying output stream, we must stub the method, or creates a real object.